.travis.yml

```diff
@@ -35,11 +36,13 @@ matrix:
       env: SYMFONY_VERSION='3.2.*'
     - php: 7.0
       env: SYMFONY_VERSION='3.3.*'
-
+    - php: 7.1
+      env: SYMFONY_VERSION='4.0.*' DEPENDENCIES='beta'
```

Comment:
Symfony 4 is stable now

Reply:
Thanks for this ping, Christian! I'll tweak it